src/components/CloudHeader/CloudHeaderListItem.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+const CloudHeaderListItem = props => {
+  const { children, className, href, isIcon, ...other } = props;
+
+  const CloudHeaderListItemClasses = classNames(
+    'cloud-header-list__item',
+    className
+  );
+
+  let itemStyles = {};
+  if (isIcon) {
```

Comment:
Could this just be toggling a class with `classNames` versus adding inline styles?

Reply:
Yeah, just didn't want to create a new class just to remove padding

Reply:
If you don't want the hassle of creating a whole stylesheet, you could think about leveraging `glamor` (https://github.com/threepointone/glamor), the library behind glamorous, to get around the inefficiencies of inline CSS (it creates a `<style>` tag and appends there).